### DISCUSSION TITLE
Method that checks if connection is alive?

### DISCUSSION THREAD

**Discussion by perzanko** [ankhers/mongodb#369] [Q&A]
Hello!

Thank you for this awesome library. It definitely does the job!

I wonder if there is any method that allow me to check if connection is still alive?

**Comment by scottmessinger** [comment]
@perzanko I haven't tried it, but I think this is the method:

https://github.com/elixir-mongo/mongodb/blob/bccfe80a59c2499077c16b9ecaa5eabb6161b919/lib/mongo/monitor.ex#L31
